lib/Database/ImportTransaction.cpp

```diff
@@ -174,7 +177,8 @@ void ImportTransaction::Implementation::addFileAssociationForProvider(IDCode pro
     lmdb::val existingValue;
     cursor.get(existingKey, existingValue, MDB_GET_CURRENT);
     const auto &existingData = *(TimestampedFileForProviderData*)existingValue.data();
-    llvm::sys::TimeValue existingModTime(existingData.Seconds, existingData.Nanoseconds);
+    llvm::sys::TimePoint<> existingModTime = std::chrono::time_point_cast<std::chrono::nanoseconds>(llvm::sys::toTimePoint(existingData.Seconds));
```

Comment:
`toTimePoint` assumes you're converting from a `time_t`.  We should only do that if we're using `toTimeT` on the other side.